app/layout.tsx

```diff
@@ -21,7 +22,12 @@ export default function RootLayout({
     <html lang="en">
       <body className={inter.className}>
         <AppRouterCacheProvider>
-          <ThemeProvider theme={theme}>{children}</ThemeProvider>
+          <ThemeProvider>
+            <AppBar />
+            <Box component="main" sx={{ my: 4 }}>
```

Comment:
@terotik @lilia1891 `Box` is a handy MUI component which just renders a div but allows you to use the `sx` styling syntax